lib/fs/debug.c

```diff
@@ -115,6 +118,45 @@ static int cmd_fs(int argc, const cmd_args *argv)
             printf("error %d unmounting device\n", err);
             return err;
         }
+    } else if (!strcmp(argv[1].str, "format")) {
+        int err;
+
+        if (argc < 3) {
+            goto notenoughargs;
+        }
+
+        err = fs_format_device(
+                argv[2].str,
+                (argc >= 4) ? argv[3].str : NULL,
+                NULL
+        );
+
+        if (err != NO_ERROR) {
+            printf("error %d formatting device\n", err);
+            return err;
+        }
+
+    } else if (!strcmp(argv[1].str, "stat")) {
```

Comment:
I'd prefer to leave it as just `stat` since it's part of the `fs` command. `fs fsstat` seems somewhat redundant. 
On another note, I think `stat` already exists as a top level command.